# zrj: add missing link against -lm in libHStext-*.so
# NOTYPE  GLOBAL DEFAULT  UND logf
# fixes textproc/hs-yaml (binutils 2.27 ld.gold is unhappy
--- text.cabal.orig	2015-07-30 14:25:03.000000000 +0300
+++ text.cabal
@@ -73,6 +73,7 @@ flag integer-simple
   description: Use the simple integer library instead of GMP
 
 library
+  extra-libraries: m
   c-sources:    cbits/cbits.c
   include-dirs: include
 
